bot/exts/moderation/infraction/infractions.py

```diff
@@ -101,11 +106,47 @@ async def purgeban(
         reason: t.Optional[str] = None
     ) -> None:
         """
-        Same as ban but removes all their messages of the last 24 hours.
+        Same as ban, but also cleans all their messages from the last hour.
 
         If duration is specified, it temporarily bans that user for the given duration.
         """
-        await self.apply_ban(ctx, user, reason, 1, expires_at=duration)
+        clean_cog: t.Optional[Clean] = self.bot.get_cog("Clean")
+        if clean_cog is None:
+            # If we can't get the clean cog, fall back to native purgeban.
+            await self.apply_ban(ctx, user, reason, 1, expires_at=duration)
```

Comment:
Explicit kwarg for readability. 
```suggestion
            await self.apply_ban(ctx, user, reason, purge_days=1, expires_at=duration)
```

Reply:
https://github.com/python-discord/bot/pull/2033/commits/0d66f826d11dcaba1115af596a9a0b6fe4c0cec8